client/src/app/core/translate/translation-parser.ts

```diff
@@ -25,26 +25,26 @@ export class OpenSlidesTranslateParser extends TranslateDefaultParser {
      * Subscribes to the custom translations and watches for updated custom translations.
      */
     public constructor(
-        organisationSettingsService: OrganisationSettingsService,
+        // organisationSettingsService: OrganisationSettingsService, // TODO: Cyclic dependency...
```

Comment:
Does the orga service needs translations? Or where is the cycle?

If it is not easy solvable, create a custom-translation-service just providing a public behaviour subject -> the orga service must actively push in it and this service can read.